site/app/templates/grading/electronic/ta_status/StatusStaticAnalysis.twig

```diff
@@ -0,0 +1,149 @@
+<div id="static-analysis" class="page-content">
+    <div>
```

Comment:
I think this would look a lot better if you added a little space in between the select x-axis box and the scatter plot.
```suggestion
    <div class="my-2">
```